src/Runtime/LambdaRuntime.php

```diff
@@ -101,17 +95,12 @@ public function processNextEvent($handler): void
         $result = null;
 
         try {
+            // PSR-15 adapter
             if ($handler instanceof RequestHandlerInterface) {
```

Comment:
Since this is the only PSR interface for handling an incoming HTTP request, Im fine with this special logic. I dont believe we should add more though.

Reply:
Makes sense. Adding Symfony support without adding it in here wouldn't be too bad anyway, it could be like your `SymfonyRequestHandler`, but it could go even further like this:

```php
<?php

use App\Kernel;
use Bref\Context\Context;
use Symfony\Component\HttpFoundation\Request;
use App\SymfonyRequestHandler;

include_once dirname(__DIR__).'/config/bootstrap.php';
$kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);

return new SymfonyHandler($kernel);
```

And the class `SymfonyHandler` would implement `Handler` and invoke the kernel with a Symfony request.